README.md

```diff
@@ -91,6 +91,7 @@ Please see our [guidelines](CONTRIBUTING.md) for contributing to the driver.
 * Bar Arnon                 https://github.com/I3arnon
 * Wan Bachtiar              https://github.com/sindbach
 * Mark Benvenuto            https://github.com/markbenvenuto
+* Jimmy Bogard              https://github.com/jbogard
```

Comment:
I think the user can be mentioned starting from this ticket since the suggested change is most of the work in the scope of this ticket

Reply:
Agreed